src/lib/actionButtons/PublishButton/PublishButton.js

```diff
@@ -22,12 +22,13 @@ export default class PublishButton extends Component {
 
   isDisabled = (formik) => {
     const record = formik.values;
-    return (
+    const r = (
       !this.draftAlreadyCreated(record) ||
-      (this.props.filesEnabled && !this.props.nubmerOfFiles) ||
+      (this.props.filesEnabled && !this.props.numberOfFiles) ||
       this.props.fileUploadOngoing ||
       formik.isSubmitting
     );
+    return r;
```

Comment:
The return of the assignment is not needed I guess...

Reply:
There is a lot of leftover code from debugging. Just a wip, so I wouldn't worry about it.

Reply:
Reminder for the redundant return assignment :)